src/web/app/components/teammates-common/student-name/student-name.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+/**
+ * Display student name with photo popover
+ */
+@Component({
+  selector: 'tm-student-name',
```

Comment:
The component name doesn't indicate that the "photo popover" feature comes in it.